.changeset/real-spoons-cough.md

```diff
@@ -1,5 +0,0 @@
----
-"@ultraviolet/form": minor
----
-
-Fix prop `type` on component `<TextInputFieldV2 />` that wasn't working properly
```

Comment:
This is weird it shouldn't be here, maybe if you rebase and push it will disappear?

Reply:
@lisalupi this is still an issue, did you try rebase on main?